src/OrchardCore/OrchardCore.Recipes.Core/Services/RecipeHarvester.cs

```diff
@@ -30,16 +33,9 @@ public class RecipeHarvester : IRecipeHarvester
             _logger = logger;
         }
 
+        /// <inheritdoc/>
         public virtual Task<IEnumerable<RecipeDescriptor>> HarvestRecipesAsync()
-        {
-            return _extensionManager.GetExtensions().InvokeAsync(HarvestRecipes, _logger);
-        }
-
-        private Task<IEnumerable<RecipeDescriptor>> HarvestRecipes(IExtensionInfo extension)
-        {
-            var folderSubPath = PathExtensions.Combine(extension.SubPath, "Recipes");
-            return HarvestRecipesAsync(folderSubPath);
-        }
+            => _extensionManager.GetExtensions().InvokeAsync(HarvestRecipes, _logger);
```

Comment:
While I personally prefer this, in OC we don't use expression-bodied methods.

Reply:
We started used from awhile in many places

Reply:
Can you point to examples?

Reply:
These are few:

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Email.Abstractions/SmtpResult.cs#L35

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Localization.Abstractions/CultureDictionaryRecordKey.cs#L33-L34

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Localization.Abstractions/CultureDictionaryRecordKey.cs#L48-L49

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Localization.Abstractions/CultureDictionaryRecordKey.cs#L54-L57

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Localization.Abstractions/CultureDictionary.cs#L40

https://github.com/OrchardCMS/OrchardCore/blob/4c5f2720a9a5f1a384b7b33160ce2babf166e0e0/src/OrchardCore/OrchardCore.Localization.Abstractions/CultureDictionary.cs#L92